R/tt_paginate.R

```diff
@@ -257,8 +254,10 @@ setMethod(
 
 
     if (NROW(content_table(tt)) > 0) {
-      cind <- indent + indent_mod(content_table(tt))
-      contdf <- make_row_df(content_table(tt),
+      ct_tt <- content_table(tt)
+      cind <- indent + indent_mod(ct_tt)
+      trailing_section_div(ct_tt) <- trailing_section_div(tt_labelrow(tt))
```

Comment:
As section_div setter is based solely on labelrow and datarow, this fixes the contentRow case